Model/ship.php

```diff
@@ -6,6 +6,7 @@ class Ship extends AbstractShip
 {
     private $jediFactor = 0;
     private $underRepair;
+    private $type;
```

Comment:
See note in AbstractShip about removing this.

Reply:
Done